frontend/src/components/filters/descriptions.js

```diff
@@ -35,14 +35,14 @@ export const SAMPLE_FILTERS = {
     label: "Container Barcode",
     recursive: true,
   },
-  projects__name: {
+  sample__projects__name: {
     type: FILTER_TYPE.INPUT,
-    key: "projects__name",
+    key: "sample__projects__name",
```

Comment:
why not use the correct field name and add it to the backend full_sample filter_fields ?